tests/integration/issue_test.go

```diff
@@ -487,6 +490,36 @@ func TestIssueRedirect(t *testing.T) {
 	req = NewRequest(t, "GET", path.Join("org26", "repo_external_tracker_alpha", "issues", "1"))
 	resp = session.MakeRequest(t, req, http.StatusSeeOther)
 	assert.Equal(t, "/"+path.Join("org26", "repo_external_tracker_alpha", "pulls", "1"), test.RedirectURL(resp))
+
+	// test to check that the PR redirection works if the issue unit is disabled
+	// repo1 is a normal repository with issue unit enabled, visit issue 2(which is a pull request)
+	// will redirect to pulls
+	req = NewRequest(t, "GET", path.Join("user2", "repo1", "issues", "2"))
+	resp = session.MakeRequest(t, req, http.StatusSeeOther)
+	assert.Equal(t, "/"+path.Join("user2", "repo1", "pulls", "2"), test.RedirectURL(resp))
```

Comment:
Why not just simply write  `"/user2/repo1/pulls/2`? Does the "path.Join" look more modern?

Reply:
1f0e11b225